config/config.go

```diff
@@ -418,6 +420,11 @@ var agentConfig = `
   hostname = ""
   ## If set to true, do no set the "host" tag in the telegraf agent.
   omit_hostname = false
+
+  ## Translator for SNMP OIDs
+  ## Valid values are "netsnmp" which runs snmptranslate and snmptable,
+  ## and "gosmi" which uses the gosmi library.
```

Comment:
Doesn't "internal" sound better for a regular user?
```suggestion
  ## and "internal" which uses the gosmi library.
```

Reply:
If we add another option that doesn't have external dependencies, "internal" will become not specific enough. I don't have plans to add another translator, but I want to keep options open.

Reply:
Okay fair enough.